recipe/meta.yaml

```diff
@@ -36,22 +35,26 @@ outputs:
     requirements:
       build:
         - {{ compiler('c') }}
-        - make
+        - {{ stdlib('c') }}
+        - bison
+        - flex
+        - pkg-config
       host:
         - {{ pin_subpackage("libfdt", exact=True) }}
+        - python {{ python }}
```

Comment:
Why is python needed here? The binaries are compiled from C, which is confirmed by the link check:
```
   INFO (dtc,bin/fdtoverlay): Needed DSO lib/libfdt.so.1 found in local/linux-64::libfdt==1.7.2=h9ecbd09_0
   INFO (dtc,bin/fdtoverlay): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_18
   INFO (dtc,bin/fdtput): Needed DSO lib/libfdt.so.1 found in local/linux-64::libfdt==1.7.2=h9ecbd09_0
   INFO (dtc,bin/fdtput): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_18
   INFO (dtc,bin/fdtdump): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_18
   INFO (dtc,bin/convert-dtsv0): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_18
   INFO (dtc,bin/fdtget): Needed DSO lib/libfdt.so.1 found in local/linux-64::libfdt==1.7.2=h9ecbd09_0
   INFO (dtc,bin/fdtget): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_18
   INFO (dtc,bin/dtc): Needed DSO lib/libyaml-0.so.2 found in conda-forge/linux-64::yaml==0.2.5=h7f98852_2
   INFO (dtc,bin/dtc): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_18
```
I think this should just be removed
```suggestion
```

Reply:
I agree both the dtc and libfdt outputs shouldn't need python. I had tried removing this before, and when doing so the templating can't find PYTHON. Do you know why that would be?

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1089416&view=logs&j=d818e662-7181-5ce9-b265-50d03742b7af&t=367317f3-9b77-59ac-9e12-e56370c98acd&l=607